wpiutil/src/main/java/edu/wpi/first/util/BUILD.bazel

```diff
@@ -0,0 +1,13 @@
+load("@rules_java//java:defs.bzl", "java_library")
+
+java_library(
+    name = "wpiutil",
```

Comment:
If we're going to stick with these small BUILD files, I think it'd be easier for everyone if we keep these at `*/src/main/java` (rather than in the middle of the Java packages), and have the target names matching the directory (so the label shorthand is useful). I'm not sure if we'd break up these packages any time soon.